src/core/message.ts

```diff
@@ -261,28 +264,25 @@ export class DefaultMessage implements Message {
     return this.isDeleted ? this.timestamp : undefined;
   }
 
-  versionBefore(message: Message): boolean {
-    // Check to ensure the messages are the same before comparing operation order
+  isOlderVersionOf(message: Message): boolean {
```

Comment:
I think having both `isOlderVersionOf` and `isNewerVersionOf` might create minor confusion if negating the comparison

`c1 = a.isNewerVersionOf(b)` is not the same as `c2 = !a.isOlderVersionOf(b)` because if `a` and `b` are different messages, `c1` will be false but `c2` will be true.

Maybe add a `@remark` not to negate those? Or maybe it's a non-issue... The docs do explain what the functions do.

Reply:
I think a remark to say don't negate them sounds good.

Reply:
Remark added